src/test/modelmanager_test.cpp

```diff
@@ -1218,13 +1222,13 @@ TEST_F(ReloadAvailableModelDueToConfigChange, ExpectReloadDueToMultipleInputShap
 }
 
 TEST_F(ReloadAvailableModelDueToConfigChange, ExpectReloadDueToShapeConfigurationChange_Named) {
-    mockModelVersionInstances = getMockedModelVersionInstances(initialVersions, *ieCore, config);
+    mockModelVersionInstances = getMockedModelVersionInstances(initialVersions, *ieCore, *ieCore_2, config);
     config.setShapes({{"A", {ovms::Mode::FIXED, {1, 3, 224, 224}}}});
     ovms::ModelManager::getVersionsToChange(config, mockModelVersionInstances, requestedVersions, versionsToStart, versionsToReload, versionsToRetire);
     EXPECT_THAT(*versionsToReload, UnorderedElementsAre(3));
 }
 
-class GetModelInstanceTest : public ::testing::Test {};
+class DISABLED_GetModelInstanceTest : public ::testing::Test {};
```

Comment:
Add short mention why disabled to know later what to fix

Reply:
+ not any observations so far about the issue

Reply:
Adding the comments in next MR: https://github.com/openvinotoolkit/model_server/pull/1040/files#diff-0fce58b2e45c71c2e99f89b90bb51a299b10390ab4eb684584cdeb46b6fdb0f3L460